src/openpgp.js

```diff
@@ -508,7 +509,7 @@ export async function generateSessionKey({ encryptionKeys, date = new Date(), en
 }
 
 /**
- * Encrypt a symmetric session key with public keys, passwords, or both at once. At least either public keys
```

Comment:
In this case, the old text was correct; `Message.encryptSessionKey` actually supports passing both public keys and passwords.

Reply:
I am not sure the existing text is correct English..? I'd write "at least one of" (I left only "Either..." in an attempt to mean the same, but indeed it can still be ambiguous whether you can pass both 😅 )

Reply:
"At least one of" sounds good :+1: